lib/modules/platform/gerrit/index.md

```diff
@@ -0,0 +1,72 @@
+# Gerrit
+
+## Supported Gerrit Versions
+
+All Gerrit 3.x Versions should be working.
+
+The current implementation uses the "hashtags" Feature from Gerrit and therefore needs a Gerrit version with [NoteDB](https://gerrit-review.googlesource.com/Documentation/note-db.html) backend.
+Gerrit `2.x` with NoteDB (only in `2.15` and `2.16`) is not tested, but could work.
+
+## Authentication
+
+<figure markdown>
+  ![Gerrit HTTP access token](/assets/images/gerrit-http-password.png){ loading=lazy }
+  <figcaption>First, create a HTTP access token for the Renovate account.</figcaption>
+</figure>
+
+Let Renovate use your HTTP access token by doing _one_ of the following:
+
+- Set your HTTP access token as a `password` in your `config.js` file
+- Set your HTTP access token as an environment variable `RENOVATE_PASSWORD`
+- Set your HTTP access token when you run Renovate in the CLI with `--password=`
+
+This user must be allowed to assign the Code-Review label with "+2" to their own changes for "automerge" to work.
+
+Remember to set `platform=gerrit` somewhere in your Renovate config file.
+
+## Renovate PR/Branch-Model with Gerrit and needed Permissions
+
+If you use the "Code-Review" label and want `automerge` working: you must set `autoApprove=true` in your Renovate config.
+Renovate will now add the _Code-Review_ label with the value "+2" to each of its "pull requests" (Gerrit-Change).
+
+<!-- prettier-ignore -->
+!!! note
+    The login must be allowed to give +2 for the Code-Review label.
+
+The Renovate option `automergeType: "branch"` makes no sense for Gerrit, because there are no branches used.
```

Comment:
I suggested a rewrite of this sentence, but it was rejected. We still need to rewrite, because `because there are no branches used.` does not say _who_ or _what_ doesn't use branches.

Open for suggestions!

Reply:
... to create pull requests.